assets/style_codebases.scss

```diff
@@ -0,0 +1,348 @@
+      background-repeat:no-repeat;
+      width: 47px;
+      height: 54px;
+      position: relative;
+      margin: auto;
+      display: block;
+    }
+  }
+}
+.cb-name {
+  font-size: 1.35em;
+  font-weight: 700;
+  margin-bottom: 10px;
+}
+.cb-desc {
+  margin-bottom: 15px;
+}
+.cb-list-none {
+  font-style: italic;
+}
+
+.cb-section-links {
+  background: var(--fpc-grey-a2);
+}
+
+
+
+
+// Codebase Detail Pages
+.cb-maturity-detail-header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  @include respond-to('small') {
+    flex-direction: column-reverse;
+    align-items: start;
+  }
+  margin-block-end: 30px;
+}
+
+.cb-maturity-detail-title {
+  background: none;
+
+  margin: 0 30px;
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  &::before {
+    content:'';
+    border-radius: 2px;
+
+    width: 10px;
+    height: 100%;
+    margin-left: -30px;
+    position: absolute;
+    display: block;
+  }
+  h1 {
+    margin-block-start: 0;
+    margin-block-end: 0;
+  }
+}
+.cb-maturity-stage {
+  font-size: 1.35em;
+  font-weight: 300;
+  border-radius: 4px;
+
+  padding: 15px 25px;
+  @include respond-to('small') {
+    margin-block-end: 20px;
+  }
+}
+
+.cb-detail-intro {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  @include respond-to('small') {
+    flex-direction: column;
+  }
+}
+.cb-detail-info {
+  //border-top: 1px solid var(--fpc-grey);
```

Comment:
These commentted out rules seem to be for development and should probably be removed.

Reply:
Agree. Removed.